reascripts/ReaSpeech/source/TranscriptExporter.lua

```diff
@@ -18,6 +18,12 @@ function TranscriptExporter:init()
 
   Logging.init(self, 'TranscriptExporter')
 
+  self.theme = ImGuiTheme.new {
+    styles = {
+      { ImGui.StyleVar_Alpha, 0 },
+    }
+  }
```

Comment:
Seems like we could apply this to all pop up windows? It'd also be nice to change the background color of the pop ups, and have one place to configure it.

Reply:
agreed. at this point this is mostly an example of implementation to work out any api/usage issues before applying everywhere.